packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -69,6 +69,48 @@ export async function exchangeDpopToken(
   });
 }
 
+// We
+const DEFAULT_LIFESPAN = 1800;
+
+/**
+ * Stores the resoure server session information in local storage, so that they
+ * can be leveraged on refresh.
+ * @param webId
+ * @param authenticatedFetch
+ * @param storageUtility
+ */
+async function setupResourceServerSession(
+  webId: string,
+  authenticatedFetch: typeof fetch,
+  storageUtility: IStorageUtility
+): Promise<void> {
+  const webIdAsUrl = new URL(webId);
+  const resourceServerIri = webIdAsUrl.origin;
+  const resourceServerResponse = await authenticatedFetch(
+    `${resourceServerIri}/session`
```

Comment:
Hhhmmm... hard-coding the `/session` endpoint here feels very wrong. Do we have a `Constant.ts` file anywhere already? Maybe leave it for now, but something to consider in general (i.e., are we hard-coding expected endpoints?).

Reply:
If this was for a proper, long-term fix, I would have approached it differently, but considering this is only a stopgap until a cleaner solution is implemented, it didn't seem too harmful.